packages/plugin-core/src/WorkspaceWatcher.ts

```diff
@@ -104,10 +118,16 @@ export class WorkspaceWatcher {
     );
   }
 
+  async onDidSaveTextDocument(document: TextDocument) {
+    if (DocUtil.isSchema(document)) {
+      await SchemaSyncService.instance().onDidSave({ document });
```

Comment:
We already have a vscode.showInformation() message popup when a schema updates successfully right (didn't look at the code too closely to see when exactly it pops up)?  If not, maybe it's worth it to add a notice to users that the schema has been updated when they click save?

Reply:
Added the message,


Note in the latest commit also removed the older SchemaWatcher (the code looked like it attempted to do schema parsing onChange but it didn't appear to be working). Also from UX experience I would argue we don't want to try to parse schemas on change since as the users are changing the schema before they save there is going to be invalid states of schema (which would trigger error messages before the user is done with their edits, and since we currently trigger error messages with pop ups it would get a bit annoying). So approaching the schemas as code seems like the right thing to do meaning attempting to compile when the file is saved, attempting to parse when the file is saved (Unless we rework how we display errors to not be done as pop ups). Also typical VSCode configuration saves the file when the user looses focus from the file.

Reply:
O - sorry, I was unclear with my comment (or perhaps positioning of this comment).  I didn't mean to imply that we should parse/toast on edit; I just wanted to check that we were properly informing a user on save.  Thanks!